docs/uninstallation/uninstallation_general_info.md

```diff
@@ -0,0 +1,2 @@
+# Uninstalling RHOAM
```

Comment:
Same question as before. What is the need for such a short document?

Reply:
same as above, ty